conjure/controllers/deploy/gui.py

```diff
@@ -110,7 +110,10 @@ def finish(single_service=None):
                                app.ui.set_footer,
                                partial(__handle_exception, "ED"))
 
-        return controllers.use('deploystatus').render(f)
+        if app.bootstrap.running:
```

Comment:
If the bootstrap has already completed by the time we get here, this will probably still be true, so we'll call bootstrapwait.render, which will flash the KITT bar for a second before the done_callback it adds is run, which will switch the view.

I think what you want here is to check `if running is True and running.done() is False`